packages/specs/source/channel-networks.md

```diff
@@ -0,0 +1,48 @@
+# Channel Networks
+
+The framework currently contains basic support for channel networks via "virtual apps". In this design, an app instance between that set of parties can have some state deposit assigned indirectly to it, without the set of parties having a direct channel containing them. Since the app instance is not directly funded, no commitment to `StateChannelTransaction.sol:executeAppConditionalTransaction` is made to that app instance. Instead, a number of commitments to `TwoPartyVirtualEthAsLump` are made in different direct channels to the same app instance (i.e., passing the same `appIdentityHash`). The commitment passes the following struct as a field.
```

Comment:
You should rephrase to define this notion. "Indirectly funded" could be _defined_ as this

Reply:
Disagree - I think it is more consistent to define "indirectly funded" as "funded but not directly funded", where "funded" is a primitive/axiomatic concept, which means that this description is a specific way that an app is "indirectly funded"

Reply:
I just mean to say we should name this specific construction of having two apps funded via two `TwoPartyVirtualEthAsLump.delegateTarget` function calls

Reply:
Let's do that in your spike for this thursday.